packages/cli/scripts/bootstrapFinder.ts

```diff
@@ -0,0 +1,84 @@
+
+import { ssz } from '@lodestar/types'
+import { ForkName } from '@lodestar/params'
+import { computeSyncPeriodAtSlot } from '@lodestar/light-client/utils'
+import { concatBytes, hexToBytes } from '@ethereumjs/util'
+
+
+const { Client } = jayson
+
+const main = async () => {
+    const beaconConfig = createBeaconConfig(defaultChainConfig, hexToBytes(genesisData.mainnet.genesisValidatorsRoot))
+    const capellaForkDigest = beaconConfig.forkName2ForkDigest(ForkName.capella)
+    const beaconNode = 'https://lodestar-mainnet.chainsafe.io/'
+    const ultralights: jayson.HttpClient[] = []
+    for (let x = 0; x < 10; x++) {
+        ultralights.push(Client.http({ host: '127.0.0.1', port: 8545 + x }))
+    }
+
+
+    console.log('Retrieving bootstrap and updates from Beacon node...')
+    let optimisticUpdate = ssz.capella.LightClientOptimisticUpdate.fromJson((await (await fetch(beaconNode + 'eth/v1/beacon/light_client/optimistic_update')).json()).data)
+    let optimisticUpdateKey = getBeaconContentKey(BeaconLightClientNetworkContentType.LightClientOptimisticUpdate, LightClientOptimisticUpdateKey.serialize({ optimisticSlot: BigInt(optimisticUpdate.attestedHeader.beacon.slot) }))
+    const currentPeriod = computeSyncPeriodAtSlot(optimisticUpdate.signatureSlot)
+    const oldPeriod = (currentPeriod - 3)
+    const updatesByRange = (await (await fetch(beaconNode + `eth/v1/beacon/light_client/updates?start_period=${oldPeriod}&count=4`)).json())
+
+    const range: Uint8Array[] = []
+    for (const update of updatesByRange) {
+        range.push(concatBytes(
+            capellaForkDigest,
+            ssz.capella.LightClientUpdate.serialize(
+                ssz.capella.LightClientUpdate.fromJson(update.data),
+            ),
+        ))
+    }
+    const serializedRange = LightClientUpdatesByRange.serialize(range)
+    const rangeKey = getBeaconContentKey(BeaconLightClientNetworkContentType.LightClientUpdatesByRange, LightClientUpdatesByRangeKey.serialize({ startPeriod: BigInt(oldPeriod), count: 6n }))
+    for (let x = 0; x < 4; x++) {
+        const bootstrapSlot = updatesByRange[x].data.finalized_header.beacon.slot
+        const bootstrapRoot = (await (await fetch(beaconNode + `eth/v1/beacon/blocks/${bootstrapSlot}/root`)).json()).data.root
+        const bootstrap = ssz.capella.LightClientBootstrap.fromJson((await (await fetch(beaconNode + `eth/v1/beacon/light_client/bootstrap/${bootstrapRoot}`)).json()).data)
+        await ultralights[Math.floor(Math.random() * 5)].request('portal_beaconStore', [getBeaconContentKey(BeaconLightClientNetworkContentType.LightClientBootstrap, LightClientBootstrapKey.serialize({ blockHash: hexToBytes(bootstrapRoot) })), toHexString(concatBytes(capellaForkDigest, ssz.capella.LightClientBootstrap.serialize(bootstrap)))])
```

Comment:
idea of randomzing here is to select from a bigger set? this seems to only go uptill node 6 out of i think 10 nodes? just observing that reselection of a node is also possible (if thats intended behavior here)

Reply:
Yep, basically just to distribute the bootstraps among several nodes to verify that the process of identifying a good bootstrap by each individual client gets a little more randomness.